aii-core/pom.xml

```diff
@@ -207,6 +207,7 @@
             <requires>
               <require>perl-CDB_File</require>
               <require>perl-XML-Simple</require>
+              <require>perl-Parallel-ForkManager</require>
```

Comment:
You may be right... In fact the real issue was https://github.com/quattor/template-library-os/pull/90 and I forgot the automatic generation of dependencies when building RPM...